Add release notes for 2020.3.1 Demeter (#91)
kasenvr/vircadia-docs-sphinx: docs/source/release-notes/2020-3-1-Demeter.md

# Alpha 2020.3.1 Demeter Release Notes

This version of Vircadia is an update to 2020.3.0. It is compatible with the 2020.3.0-Demeter protocol. This version was released on October 24th, 2020.

## Interface (Codename Athena)

The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).

#### General

> * Application: Update shortcut names to include the "Vircadia" in them. (#651)

Reviewer: ```suggestion
* Application: Update shortcut names to include "Vircadia" in them. (#651)
```

> * UI: Add a checkbox for more details to messages logged in entity scripts such as the scriptURL and function with line numbers. (#669)

Reviewer: ```suggestion
* UI: Add a checkbox for more details to messages logged in entity scripts such as the script URL and function with line numbers. (#669)
```

* Application: Allow mouse capture via scripting which enables first person and third person shooter camera views on desktop. (#677)
* App: 'Create' app now uses local entities instead of deprecated 3D overlays. (#737)
* App: 'Create' app now has: Copy, Cut, Paste, and Duplicate buttons for VR; Undo and Redo buttons for desktop and VR. (#760)
* Graphics: Fix handling of missing glTF blendshape target meshes. (#773)
* API: Add `url` parameter to `Window.openWebBrowser`. (#778)
* App: Add script that prompts a user to enter a display name on first run if one is not set. (#805)
* Application: Add 'Valve Index' workaround to allow calibration for full body tracking. (#808)
* App: 'Explore' app now references 'Location Name' when adding a location. (#814)

#### Miscellaneous

* Application: Fix build warnings. (#761)

> * Application: Set GL preference to 'GLVND' from 'LEGACY' then revert back to 'LEGACY'. (#763, #806)

Reviewer 2: It went from undefined to GLVND and then to LEGACY again. =P

* App: Fix CDN references to hifi-production in tutorial example scripts. (#788)

## Domain Server

The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).

#### General

* Server: Add wizard step to enable/disable automatic threading for the avatar and audio mixers. (#546)
* Server: Hide temp place name button, places table, and places wizard step. Add step during account connection to label the domain. (#803)

## Metaverse Server & Dashboard (Codename Iamus)

[Iamus](https://github.com/kasenvr/iamus) Metaverse Server updates can be found [here](https://github.com/kasenvr/Iamus/commits/master).

[Iamus Metaverse Dashboard](https://github.com/vircadia/project-iamus-dashboard) updates can be found [here](https://dashboard.vircadia.com/).

## Vircadia Launcher (Codename Pantheon)

[Vircadia's launcher](https://github.com/vircadia/pantheon-launcher) updates can be found [here](https://github.com/vircadia/pantheon-launcher/commits/master). Some changes are listed below. The launcher is now on experimental version 0.0.1s.

* Fix launching of Interface while the launcher is running as an admin.

> * Fix sandbox closing if the launcher is closed.

Reviewer: ```suggestion
* Fix Sandbox closing if the launcher is closed.
```
